app/scripts/DragListeningDiv.js

```diff
@@ -18,7 +18,13 @@ class DragListeningDiv extends React.Component {
 
   render() {
     // color red if not enabled, green if a track is not top and red otherwise
```

Comment:
Very minor: This comment seems outdated not that there is blue.

Reply:
Good catch. Fixed.